unimport/console.py

```diff
@@ -1,11 +1,63 @@
+import argparse
 import os
-import sys
+import pathlib
 import tokenize
 
+from unimport.config import Config
 from unimport.files import get_files
 from unimport.unused import get_unused
 
 
+class Cli(object):
```

Comment:
`object` as a base is unnecessary

Reply:
Also naming can be adjusted to `CLI` instead of `Cli`, which looks better for a shorthand.